frontend/components/dashboard/table/SummaryStatus.vue

```diff
@@ -0,0 +1,113 @@
+<script setup lang="ts">
+import type { VDBSummaryStatus } from '~/types/api/validator_dashboard'
+import type { SlotVizIcons } from '~/types/dashboard/slotViz'
+
+// TODO: replace with v2.5 summary data
+interface Props {
+  status: VDBSummaryStatus
+}
+const props = defineProps<Props>()
+
+const { t: $t } = useI18n()
+
+const mapped = computed(() => {
+  const mapCount = (count: number, key: string, icon: SlotVizIcons) => {
+    return {
+      icon,
+      key,
+      className: count > 0 ? `${key.replace('_', '-')} active` : '',
+      tooltip: $t(`dashboard.validator.summary.status.${key}`, {}, count)
+    }
+  }
+
+  // TODO: replace with v2.5 logic (once we got the api structs)
+  const scheduledSyncCount = props.status?.next_sync_count
+  const currentSyncCount = props.status?.current_sync_count
+  const slashedCount = props.status?.slashed_count
```

Comment:
On lines 24, 25 and 26, the `?` is ineffective because line `7` makes `prop.status` mandatory.